site/templates/film/item.jet

```diff
@@ -157,9 +157,7 @@
               <h2>{{i18n("meta_detail_bonus_title")}}</h2>
               {{yield list(itemsPerRow=4, itemLayout="landscape") content}}
                 {{range bonus := film.Bonuses}}
-                  <div class="sub-item">
-                    {{yield subItem(item=bonus)}}
-                  </div>
+                  {{yield subItem(item=bonus)}}
```

Comment:
The sub-item wrapper div that you've removed here also needs to be removed from the tv detail jet (episodes and bonuses)!

Reply:
Ah crap good catch. Yes removed also.